ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -281,29 +281,31 @@ def ocs_install_verification(
     # TODO: Verify ceph osd tree output have zone or rack based on AZ
 
     # Verify CSI snapshotter sidecar container is not present
-    log.info("Verifying CSI snapshotter is not present.")
-    provisioner_pods = get_all_pods(
-        namespace=defaults.ROOK_CLUSTER_NAMESPACE,
-        selector=[
-            constants.CSI_CEPHFSPLUGIN_PROVISIONER_LABEL,
-            constants.CSI_RBDPLUGIN_PROVISIONER_LABEL
+    # if the OCS version is < 4.6
+    if float(config.ENV_DATA['ocs_version']) < 4.6:
+        log.info("Verifying CSI snapshotter is not present.")
+        provisioner_pods = get_all_pods(
+            namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+            selector=[
+                constants.CSI_CEPHFSPLUGIN_PROVISIONER_LABEL,
```

Comment:
we have not supported alpha snapshot container for cephfs provisioner pod. it always passes

Reply:
yeah, it was passing in their run also but I kept in under the same check just to be in safe side. We can remove it also.

The only check which failed was assert {'name': 'CSI_ENABLE_SNAPSHOTTER', 'value': 'false'}